doc/source/building/build_live_iso.rst

```diff
@@ -1,44 +1,45 @@
 Build an ISO Hybrid Live Image
 ==============================
 
-.. hint::
+.. sidebar:: Abstract
 
```

Comment:
I rebuild the docs with '.. container:: Abstract' and to be honest, sidebar looks much better :)
The sidebar also better separates the abstract information from the text. So let's go with the sidebar